src/app/modules/core/components/user-profile/contact-list/contact-list.component.html

```diff
@@ -0,0 +1,61 @@
+<div class="container">
+  <div class="col">
+    <mat-form-field>
+      <mat-label>Filter</mat-label>
+      <input matInput (keyup)="applyFilter($event)" placeholder="John" #input>
+    </mat-form-field>
+
+    <div class="mat-elevation-z8">
+      <table mat-table [dataSource]="dataSource" matSort>
+
+        <!-- ID Column -->
+        <ng-container matColumnDef="id">
```

Comment:
1) Individual table column can be made re-usable component if it is placed inside a separate component for table-column.
2) Rather than mentioning each column here, we can iterate over Table column headings and render column components. which will help if we need to add extra columns in future (just need to change the header array and result gets reflected on table)